reactor-core/src/main/java/reactor/core/publisher/Flux.java

```diff
@@ -7899,6 +7899,51 @@ public final void subscribe(Subscriber<? super T> actual) {
 		return subscriber;
 	}
 
+	/**
+	 * Provide current {@link Flux<T>} transformation when the first available signal
```

Comment:
replace the next two entire paragraphs with the following suggestion

```
	 * Transform the current {@link Flux<T>} once it emits its first element, making a
	 * conditional transformation possible. This operator first requests one element
	 * from the source then applies a transformation derived from a {@link Signal}.
	 * Note that the source might complete or error immediately instead of emitting,
	 * in which case the {@link Signal} would be onComplete or onError. It is NOT
	 * necessarily an onNext Signal, and must be checked accordingly.
```

Reply:
done